models/action.go

```diff
@@ -238,7 +239,7 @@ func (a *Action) getCommentLink(e Engine) string {
 
 // GetBranch returns the action's repository branch.
 func (a *Action) GetBranch() string {
-	return a.RefName
+	return strings.TrimPrefix(git.BranchPrefix, a.RefName)
```

Comment:
```suggestion
	return strings.TrimPrefix(a.RefName, git.BranchPrefix)
```

Reply:
that's what I get for doing things quickly without checking them!